gitarro.rb

```diff
@@ -8,6 +8,8 @@
 require_relative 'lib/gitarro/backend'
 
 b = Backend.new
+exit 0 if b.triggered_by_pr_number?
```

Comment:
why are we moving this outside?

Reply:
If we call the command with a -P parameter, we don't need to checkout all the PRs from github and jump in an iterator, as we can just checkout directly the PR passed by param.

Reply:
ok